src/qml/AboutPage.qml

```diff
@@ -29,6 +29,26 @@ Flickable {
         id: diskUsage
     }
 
+    Item {
+        id: uptimeCounter
+        property int totaltime : uptime
+
+        Timer {
+            interval: 1000
+            repeat: true
+            triggeredOnStart: true
+            running: true
+            onTriggered: ++uptimeCounter.totaltime
+        }
+        function asString() {
+            var now = uptimeCounter.totaltime
+            var days = Math.floor(now / 86400)
+            var date = new Date(1000 * (now - days * 86400))
+            var timeString = date.toISOString().substring(11, 19);
+            return `${days} d ${timeString}`
```

Comment:
I think the d for days should be written out as days and be a translation string.

Reply:
Good point!  I've modified it to do that.